core/src/main/java/org/radargun/stats/BasicStatistics.java

```diff
@@ -28,6 +28,9 @@
    @Property(name = "operationStats", doc = "Operation statistics prototype.", complexConverter = OperationStats.Converter.class)
    protected OperationStats prototype = new BasicOperationStats();
 
+   @Property(doc = "Operation per second.")
+   public int opsPerSec = 0;
```

Comment:
I don't think this should be configured in statistics; it should be a property of the benchmark stage and this class would have just the setter.
Ideally, `opsPerSec` would be computed in the stage based on the benchmark setup (which usually sets per-operation rate or ratio).